### PR TITLE
Adding go to definition and removing edit function from calls

### LINKED ISSUE #2771: Go to Definition
**Describe the bug**
When a program is large enough, it is impossible to find the definition of a function. Go to definition in the content menu would go a long way.

The call block has "edit function" which should go replaced by "Go to Function"
image

### PR DESCRIPTION
**Pull request by riknoll**
Fixes https://github.com/microsoft/pxt-microbit/issues/2771

This adds a "go to definition" option on the context menu for function calls and removes "edit function" so that now it's only on definitions.

### CODE REVIEW

File: core/pxt_blockly_functions.js [hunk 1]
```diff
@@ -395,6 +395,19 @@ Blockly.Functions.makeEditOption = function(block) {
   return editOption;
 };
 
+Blockly.Functions.makeGoToDefinitionOption = function(block) {
+  var gtdOption = {
+    enabled: true,
+    text: Blockly.Msg.FUNCTIONS_GO_TO_DEFINITION_OPTION,
+    callback: function() {
+      var functionName = block.getField("function_name").getText();
```
Comment (shakao) [inline core/pxt_blockly_functions.js:403]: should this be getValue?

Comment (riknoll) [reply core/pxt_blockly_functions.js:403]: should it? I just copied what we do in blocklycompiler

Comment (shakao) [reply core/pxt_blockly_functions.js:403]: HMM no i think you're right, gettext is safer for compatibility with old functions

**Review by shakao — APPROVED** [review]

**Comment by abchatra** [comment]
@tballmsft  for you

**Pull request merged**